symplyphysics/laws/dynamics/damped_oscillations/q_factor_via_bandwidth.py

```diff
@@ -0,0 +1,51 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    dimensionless,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Q factor (quality factor) is a dimensionless parameter that describes how underdamped 
+## an oscillator or resonator is: the larger the Q factor is, the less damped it is. There 
+## are two nearly equivalent definitions of it that become approximately equivalent as Q 
+## becomes larger, meaning that the resonator becomes less damped.
+
+# Law: Q = f_r / delta_f
+## Q - Q factor
+## f_r - oscillator's resonant frequency
+## delta_f - resonance width, or full width at half maximum, i.e. the bandwidth over which
+##           the power of vibration is greater than half the power at the resonant frequency
```

Comment:
It's very hard to embrace, what it is. Do you have any better description of `resonance_width`?

Reply:
I'll write a more clear explanation